data/crac/crac-util/src/test/java/com/farao_community/farao/data/crac_util/CracCleanerTest.java

```diff
@@ -0,0 +1,156 @@
+/*
+ *  Copyright (c) 2020, RTE (http://www.rte-france.com)
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_util;
+
+import com.farao_community.farao.commons.Unit;
+import com.farao_community.farao.data.crac_api.*;
+import com.farao_community.farao.data.crac_impl.SimpleCrac;
+import com.farao_community.farao.data.crac_impl.range_domain.Range;
+import com.farao_community.farao.data.crac_impl.range_domain.RangeType;
+import com.farao_community.farao.data.crac_impl.remedial_action.network_action.ComplexNetworkAction;
+import com.farao_community.farao.data.crac_impl.remedial_action.network_action.Topology;
+import com.farao_community.farao.data.crac_impl.remedial_action.range_action.PstWithRange;
+import com.farao_community.farao.data.crac_impl.threshold.AbsoluteFlowThreshold;
+import com.farao_community.farao.data.crac_impl.threshold.RelativeFlowThreshold;
+import com.farao_community.farao.data.crac_impl.usage_rule.FreeToUse;
+import com.powsybl.iidm.network.Network;
+import com.farao_community.farao.data.crac_impl.utils.NetworkImportsUtil;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+/**
+ * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
+ */
+
+public class CracCleanerTest {
```

Comment:
Add tests by varying results when varying features

Reply:
I've corrected everything except here, for some reason, the test fails if I uncomment the commented test, and test the whole file. But when I test each test class individually, it works. I will fix this on Monday.

Reply:
Done.